gcloud/bigtable/cluster.py

```diff
@@ -74,6 +85,73 @@ def _prepare_create_request(cluster):
     )
 
 
+def _pb_timestamp_to_datetime(timestamp):
+    """Convert a Timestamp protobuf to a datetime object.
+
+    :type timestamp: :class:`._generated.timestamp_pb2.Timestamp`
+    :param timestamp: A Google returned timestamp protobuf.
+
+    :rtype: :class:`datetime.datetime`
+    :returns: A UTC datetime object converted from a protobuf timestamp.
+    """
+    return (
+        _EPOCH +
+        datetime.timedelta(
+            seconds=timestamp.seconds,
+            microseconds=(timestamp.nanos / 1000.0),
+        )
+    )
```

Comment:
I've been going back on forth on that. Do you have a reason to put them one place or another? I can't decide.

-----

The https://github.com/dhermes/gcloud-python-bigtable is "done" and the only place these helpers ended up being used were `cluster.py`. They actually live in `gcloud_python._helpers` over there, so I moved them to have them closer to the place they are used. Their existence is just for "code quality" and making it easier to test things.